packages/buttons/src/elements/ToggleButton.tsx

```diff
@@ -11,17 +11,16 @@ import Button, { IButtonProps } from './Button';
 
 export interface IToggleButtonProps extends IButtonProps {
   /**
-   * Determine if the button is pressed or not. Use `'mixed'` to indicate
-   * whether the toggle controls other elements which do not share the same
-   * value.
+   * Determines if the button is pressed. Use "mixed" to indicate that
+   * the toggle controls other elements which do not share the same value
```

Comment:
Per content [guidelines](https://garden.zendesk.com/content/punctuation#period) the second sentence should end with a period.